packages/lit-element/docs/404.html

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I would remove all of the docs for the initial commit.

Reply:
Done.